modules/nuxt-ssr-lit.yml

```
@@ -0,0 +1,20 @@
+name: ssr-lit
+description: Server-Side Rendering for Lit Element components
+repo: prashantpalikhe/nuxt-ssr-lit
+npm: nuxt-ssr-lit
+icon: ''
+github: https://github.com/prashantpalikhe/nuxt-ssr-lit
+website: https://github.com/prashantpalikhe/nuxt-ssr-lit
+learn_more: ''
+category: Devtools
```

Should this be Libraries?

```suggestion
category: Libraries
```

---

Yes, it suits it a little better